src/pages/Vote/index.tsx

```diff
@@ -135,6 +139,13 @@ export default function Vote() {
     uniBalance && JSBI.notEqual(uniBalance.quotient, JSBI.BigInt(0)) && userDelegatee === ZERO_ADDRESS
   )
 
+  // show create proposal button if they pass the proposal threshold
```

Comment:
I wonder if we should always show the button but disabled. As a way to upsell governance.

Reply:
Yep, Hayden mentioned this in Slack. Fixed in my local branch, haven't push the commits.